seshdash/tests/test_settings_buttons.py

```diff
@@ -0,0 +1,101 @@
+                                             system_voltage=24,
+                                             number_of_panels=12,
+                                             vrm_site_id=213,
+                                             battery_bank_capacity=12321,
+                                             has_genset=True,
+                                             has_grid=True)
+
+        self.data_point = Data_Point.objects.create(site=self.site,
+                                                    soc=10,
+                                                    battery_voltage=20,
+                                                    time=timezone.now(),
+                                                    AC_input=0.0,
+                                                    AC_output=15.0,
+                                                    AC_Load_in=0.0,
+                                                    AC_Load_out=-0.7)
+        #create sesh rmc account
+        self.test_rmc_account = Sesh_RMC_Account(api_key='lcda5c15ae5cdsac464zx8f49asc16a')
+        self.test_rmc_account.save()
+
+        #create rmc status
+        self.test_rmc_status = RMC_status.objects.create(rmc=self.test_rmc_account,
+                                                        ip_address='127.0.0.1',
+                                                        minutes_last_contact=100,
+                                                        signal_strength=27,
+                                                        data_sent_24h=12,
+                                                        time=datetime.now())
+        self.test_rmc_status.save()
+
+        #create test user
+        self.test_user = User.objects.create_user("patrick", "alp@gle.solar", "cdakcjocajica")
+        self.test_sesh_user = Sesh_User.objects.create(user=self.test_user,phone_number='250786688713' )
+        #assign a user to the sites
+
+
+        assign_perm("view_Sesh_Site",self.test_user,self.site)
+
+        generate_auto_rules(self.site.pk)
+
+
+    #testing form
+    def test_form(self):
+        f = Client()
+        f.login(username = "patrick",password = "cdakcjocajica")
+        data={'site_name':u'kibuye',
+                              'comission_date':datetime(2015,12,11,22,0),
+                              'location_city':u'kigali',
+                              'location_country':u'rwanda',
+                              'time_zone':'Africa/Kigali',
+                              'position_0':36,
+                              'position_1':42,
+                              'installed_kw':2,
+                              'system_voltage':4,
+                              'number_of_panels':100,
+                              'battery_bank_capacity':1200}
+        form = SiteForm(data)
+        self.assertTrue(form.is_valid())
+        form.save()
+        response = f.post('/add_site', data)
+        self.assertEqual(response.status_code, 200)
```

Comment:
This test is incomplete are you checking that the site was actually created in the database?